acquia/acquia_cms
# ACMS-918: Backstop tests failing as event url is changed now.

**Motivation**

Fixes Backstop failing tests.

**Proposed changes**
Backstop tests url changed as they now no longer contains `past` word in the title as well in URL alias.

**Testing steps**

- Install Site using Site studio.
- Enable Acquia CMS starter module.
- Run command: `npm install &&  npm run backstop-starter`
- Backstop tests should pass.

**Merge requirements**
- [ ] _Minor change_ applied
